glean-core/ios/Glean/Glean.swift

```diff
@@ -498,5 +492,6 @@ public class Glean {
         // Init Glean.
         testDestroyGleanHandle()
         initialize(uploadEnabled: true, configuration: configuration)
+        Dispatchers.shared.concurrentOperationsQueue.waitUntilAllOperationsAreFinished()
```

Comment:
I understand this is so that it effectively becomes a synchronous init when testing.
Do we do something similar on the Kotlin side as well?

Reply:
I had originally put this here to try and solve an issue with a particular test that broke with the async changes.  I've now determined that this wait is no longer necessary so I'll remove it.